Tanasee/Tanasee1.xml

```diff
@@ -35,14 +38,684 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <desc type="LandGrant">Note written by a hand similar to that of the previous <ref target="#a25 #a26"/>, but taller 
+                                        and written with an irregular margin to the right on a land grant in the time of <persName ref="PRS6229LebnaDe"/>.</desc>
+                                </item>
+                                <item xml:id="a28">
+                                    <locus target="#237ra"/>
+                                    <desc type="LandGrant"/>
+                                </item>
+                                <item xml:id="a29">
+                                    <locus target="#237va"/>
+                                    <desc type="LandGrant">Land grant written in very small characters and partly not legible in the microfilm images: 
+                                        <foreign xml:lang="gez">ኍልቈ፡ ምድር፡ ዘወራሚት፡ ቢኖራ፡ ፪ምድር፡ ዘ፩ምድር፡ 
+                                            <gap reason="ellipsis" unit="lines" quantity="19"/></foreign></desc>
+                                </item>
+                                <item xml:id="a30">
+                                    <locus target="#237vb"/>
+                                    <desc type="LandGrant">Land grants issued by a king named <foreign xml:lang="gez">አድያም፡ ሰገድ፡</foreign>, who can be 
+                                        identified as either <persName ref="PRS5616IyasuI"/>, <persName ref="PRS5617IyasuII"/> or 
+                                        <persName ref="PRS5636IyoasI"/>.</desc>
+                                </item>
+                                <item xml:id="a31">
+                                    <locus target="#237vb"/>
+                                    <desc type="RecordDistribution">Possibly a record of taxes to be distributed to a number of local chiefs.</desc>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#2r"/>
+                                    <desc type="OwnershipNote"><foreign xml:lang="gez">ዘደሴት፡ ቅዱስ፡ ገብርኤል፡ መጽሐ።</foreign> by a modern Ethiopian 
+                                        hand on top of the folio with previously erased text.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#4va"/>
+                                    <desc type="findingAid">List of Biblical books: <foreign xml:lang="gez">ኦሪት፡ ዮዲት፡ ሕዝቅኢል፡ ዮዲት፡ አስቴር፡ በ፡ ኩፋሌ፡ ዳዊት፡ 
+                                        ጥበበ፡ ሰሎሞን፡ </foreign></desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#85va"/>
+                                    <desc type="DonationNote">Donation note covering 11 lines of the first column with a reference to 
```

Comment:
I would class all these notes to which you were able to assign a clear type as additions

Reply:
OK.